server/src/main/scala/za/co/absa/atum/server/api/http/Endpoints.scala

```diff
@@ -47,4 +47,14 @@ trait Endpoints extends BaseEndpoints {
       .out(jsonBody[AtumContextDTO])
   }
 
+  protected val createOrUpdateAdditionalDataEndpoint
+    : PublicEndpoint[AdditionalDataSubmitDTO, ErrorResponse, AdditionalDataSubmitDTO, Any] = {
+    apiV1.post
+      .name(CreateOrUpdateAdditionalData)
+      .in(CreateOrUpdateAdditionalData)
+      .in(jsonBody[AdditionalDataSubmitDTO])
+      .out(statusCode(StatusCode.Created))
```

Comment:
Or StatusCode.Ok? I am not sure about this one and maybe we can revisit this at some later point, but I am curious how others see it. @benedeki @TebaleloS

Reply:
Let's decide on all this during API v2 phase - but to me it doesn't matter that much, no one will really see the difference / utilize this information. 

But if we really want to dig deeper, this is a very good conversation about it: https://stackoverflow.com/questions/1860645/create-request-with-post-which-response-codes-200-or-201-and-content

I like this opinion:
* 200 when an object is created and returned
* 201 when an object is created but only its reference is returned (such as an ID or a link)

So, for APIs that existed in V1, I wouldn't change anything to make it backward compatible, and then in general for those 3 endpoints, for create partitioning, checkpoint creation, and AD creation I think that 200 makes sense (so far)